ansible_rulebook/websocket.py

```diff
@@ -97,13 +98,23 @@ async def send_event_log_to_websocket(
         logger.info("websocket %s connected", websocket_address)
         event = None
         try:
+            if last_payload:
+                logger.info("Resending last event...")
+                await websocket.send(json.dumps(last_payload))
+                last_payload = None
+
             while True:
-                event = await event_log.get()
+                last_payload = event = await event_log.get()
```

Comment:
@hsong-rh can we just use event instead of the last_payload

Reply:
Changed